npm-js/create-kalix-entity/template/event-sourced-entity/proto/counter_api.proto

```diff
@@ -30,9 +30,13 @@ message CurrentCounter {
 }
 
 service CounterService {
-  option (kalix.service) = {
-    type : SERVICE_TYPE_ENTITY
-    component : ".domain.Counter"
+  option (kalix.codegen).event_sourced_entity = {
+    name: "Counter"
+    entity_type: "counter"
+    state: ".domain.CounterState"
+    events: ".domain.ValueIncreased"
+    events: ".domain.ValueDecreased"
+    events: ".domain.ValueReset"
```

Comment:
We may want to use fully qualified names here, as the relative ones have tripped people before when they've also used an `api` package. I thought we were doing that consistently for JVM SDKs, but maybe not.